StudCity.Frontend/src/app/features/register-complete/registerCompleteSlice.js

```diff
@@ -11,7 +11,7 @@ const initialState = {
 		phoneNumber: ""
 	},
 	avatar: {},
-	role: {},
+	role: 0,
```

Comment:
why don't you use string instead of a number, I think using a strings like 'teacher' and 'student' more intuitively understandable